docs/guides/building/cross-platform.md

```diff
@@ -149,10 +151,104 @@ env:
 
+## Experimental: Build Windows apps on Linux and macOS
+
+Tauri v1.3 added a new Windows installer type based on the [NSIS] installer framework. In contrast to WiX, NSIS itself can also work on Linux and macOS which makes it possible to build many Tauri apps on non-Windows hosts. Note that this is currently considered highly experimental and may not work on every system or for every project. Therefore it is only recommended if CI solutions like GitHub Actions or local VMs don't work for you.
+
+Since Tauri officially only supports the msvc Windows target and not the gnu one, the setup is a bit more involved.
+
+First, make sure all your Tauri dependencies are at least version 1.3, check out the [dependency update guide] if you're not sure how.
+
+#### Install NSIS
+
+Some Linux distributions have NSIS available in their repositories, for example on Ubuntu you can install NSIS by running this command:
+
+```sh title=Ubuntu
+sudo apt install nsis
+```
+
+But on many other distributions you have to compile NSIS yourself or download Stubs and Plugins manually that weren't included in the distro's binary package. Fedora for example only provides the binary but not the Stubs and Plugins:
+
+```sh title=Fedora
+sudo dnf in mingw64-nsis
+wget https://github.com/tauri-apps/binary-releases/releases/download/nsis-3/nsis-3.zip
+unzip nsis-3.zip
+sudo cp nsis-3.08/Stubs/* /usr/share/nsis/Stubs/
+sudo cp -r nsis-3.08/Plugins/** /usr/share/nsis/Plugins/
+```
+
+On macOS you will need [Homebrew] to install NSIS:
+
+```sh title=macOS
+brew install nsis
+```
+
+#### Install LLVM and the LLD Linker
+
+Since the default Microsoft linker only works on Windows we will also need to install a new linker. To compile the Windows Resource file which is used for setting the app icon among other things we will also need the `llvm-rc` binary which is part of the LLVM project.
+
+```sh title="Ubuntu"
+sudo apt install lld llvm
+```
+
+```sh title=macOS
+brew install llvm
+```
+
+On macOS you also have to add `/opt/homebrew/opt/llvm/bin` to your `$PATH` as suggested in the install output.
+
+#### Install the Windows Rust target
+
+Assuming you're building for 64-bit Windows systems:
+
+```sh
+rustup target add x86_64-pc-windows-msvc
+```
+
+#### Install the Windows SDKs
+
+To get the Windows SDKs required by the msvc target we will use the [xwin] project:
+
+```sh
+cargo install xwin
+```
+
+Then you can use the `xwin` CLI to install the needed files to a location of your choice. Remember the location, we will need it in the next step. In this guide we will create a `xwin` directory in the Home directory.
+
+```sh
+xwin --accept-license splat --disable-symlinks --output ~/xwin
+```
+
+If you want to read Microsoft's license upon installation, you can run the same command without the `--accept-license` flag.
```

Comment:
Could we change this the other way around, eg if you want to accept all licenses at once, use this flag.

Reply:
Actually, let's completely remove it, it's just one simple yes/no prompt.